Individual.cpp

```diff
@@ -216,5 +217,31 @@ ostream& operator<<(ostream &fout, Individual& ind)
 	return fout;
 }
 
+void Individual::setIndividualMatchFile()
+{
+	string ext = ".tsv";
+	string dir = "individual_match_files/";
```

Comment:
Folder should be named `dog_level_match_files` to preserve behaviour of existing rust code?

Reply:
(I know you mentioned you still needed to tweak these names, so calling out what I think based on my time with trust code)